mkosi

```diff
@@ -4307,6 +4312,7 @@ def build_stuff(args: CommandLineArguments) -> None:
     if root_hash is not None:
         print_step(f'Root hash is {root_hash}.')
 
+    del dir_fd
```

Comment:
Shouldn’t we `os.close(fd)` instead? As far as I can tell, `os.open` returns a plain `int`, so deleting it won’t have any special effect.

Reply:
indeed